fullstop/cli.py

```diff
@@ -183,6 +183,9 @@ def format_meta_info(meta_info):
         return ''
     if isinstance(meta_info, str):
         return meta_info
+    # remove application properties from meta_info, as they are now separate columns in the output table
+    meta_info.pop('application_id', None)
+    meta_info.pop('application_version_id', None)
```

Comment:
my violations have the property "application_version" and not "application_version_id"...